tests/python/pants_test/option/util/fakes.py

```diff
@@ -109,7 +109,12 @@ def items(self):
     def scope_to_flags(self):
       return {}
 
-    def get_fingerprintable_for_scope(self, scope, include_passthru=False):
+    def get_fingerprintable_for_scope(self, scope, include_passthru=False,
+                                      fingerprint_key=None, invert=False):
+      if fingerprint_key is not None:
+        raise Exception('fingerprint key not none: {}'.format(fingerprint_key))
+      if invert is True:
+        raise Exception('invert is true')
```

Comment:
1. curious why these assertions are needed
2. nit: `if invert` instead of `if invert is True`

Reply:
`fingerprint_key` and `invert` were added here to maintain compatibility with `Options#get_fingerprintable_for_scope()`, which is called from within `OptionsFingerprinter.combined_options_fingerprint_for_scope()`. However, it doesn't use either of them, so I thought it would make sense to ensure nobody tries to use them here. I decided to remove them in 7f2cd834b, as it doesn't really seem to make a lot of sense to do that.